go.mod

```diff
@@ -173,3 +173,5 @@ require (
 )
 
 replace github.com/kubernetes/cri-api => k8s.io/cri-api v0.23.5-rc.0
+
+replace github.com/aquasecurity/tracee/types v0.0.0-20230816112254-7c722620761f => ./types
```

Comment:
will be changed once this is merged https://github.com/aquasecurity/tracee/pull/3395